crates/sui-config/src/builder.rs

```diff
@@ -70,7 +70,7 @@ impl<R: ::rand::RngCore + ::rand::CryptoRng> ConfigBuilder<R> {
     pub fn build(mut self) -> NetworkConfig {
         let validators = (0..self.committee_size.get())
             .map(|_| get_key_pair_from_rng(&mut self.rng).1)
-            .map(|key_pair| ValidatorGenesisInfo {
+            .map(|key_pair: AuthorityKeyPair| ValidatorGenesisInfo {
```

Comment:
is this needed again?

Reply:
Yes, it is require - we need to specify which key type we generate in `get_key_pair_from_rng`. Although this can also be done in the function just a line up.